### PR TITLE
generator: strip first ignore-annotation in Rust

### PR DESCRIPTION
**Pull request by senekor**
[no important files changed]

### CODE REVIEW

File: rust-tooling/generate/src/lib.rs [hunk 1]
```diff
@@ -121,7 +121,13 @@ fn generate_tests(slug: &str, fn_names: Vec<String>) -> String {
...
 
     let rendered = template.render("test_template.tera", &context).unwrap();
-    let rendered = rendered.trim_start();
+
+    // Remove ignore-annotation on first test.
+    // This could be done in the template itself,
+    // but doing it here makes all templates more readable.
+    // Also, it is harder to do this in the template when the template
+    // generates test functions inside a macro for modules.
+    let rendered = rendered.replacen("#[ignore]\n", "", 1);
```
Comment (senekor) [inline rust-tooling/generate/src/lib.rs:130]: This is the interesting part - the rest is just removing the equivalent logic from all tera templates.

File: exercises/practice/allergies/tests/allergies.rs [hunk 2]
```diff
@@ -335,7 +335,6 @@ fn allergic_to_everything_cats() {
 
 #[test]
 #[ignore]
-
```
Comment (senekor) [inline exercises/practice/allergies/tests/allergies.rs:338]: The previous version of the template for `allergies` seems to have generated unnecessary whitespace. This change fixes that (by accident).

Comment (ErikSchierboom) [reply exercises/practice/allergies/tests/allergies.rs:338]: Funnily enough, I just noticed that yesterday!

**Review by ErikSchierboom — APPROVED** [review]
